NEWS

```diff
@@ -106,24 +109,13 @@ Bug fixes
 * Fix hash keys being left behind by bcf_hdr_remove.
   (PR#1535, fixes #1533.  Reported by Giulio Genovese in #842)
 
-* Added return on an error to bgzf_read_block.
-  (PR#1529, fixes #1528.  Thanks to Bergur Ragnarsson)
-
 * Make bcf_hdr_idinfo_exists more robust by checking id value exists.
   (PR#1544, fixes #1538.  Reported by Giulio Genovese)
 
-* Catch errors from bgzf_getline in hts_readlist and hts_readlines.  There were
-  a couple of places where the returned error was ignored.
-  (PR#1554)
-
-* Fix a bug in the codec learning algorithm for TOKA.  The name tokeniser has a
-  rANS vs Arithmetic coder choice as a parameter (in the "strat" variable). We
-  lacked this distinction when learning which method works best, so in the
-  choice of toka (tok3+arith) vs bzip2 vs gzip etc we selected tok3 and switched
-  back to strat 0, disabling the arithmetic coder.his only affects archive mode,
-  or where requested eg "samtools view -O cram,version=3.1,use_arith".
-  (PR#1559)
-
+* CRAM improvements. Fixed crash with multi-threaded CRAM.Fixed a bug in the
```

Comment:
"multi-threaded CRAM.Fixed a bug" - missing space after full-stop.  (OK two spaces, but that'll be my ancient brain.)